include/ibamr/IBFESurfaceMethod.h

```diff
@@ -300,6 +309,16 @@ class IBFESurfaceMethod : public IBStrategy
         const std::vector<SAMRAI::tbox::Pointer<SAMRAI::xfer::CoarsenSchedule<NDIM> > >& u_synch_scheds,
         const std::vector<SAMRAI::tbox::Pointer<SAMRAI::xfer::RefineSchedule<NDIM> > >& u_ghost_fill_scheds,
         double data_time);
+    /*!
+     * Compute the exterior fluid traction for when using the
+     * jump conditions
+     */
+    void computeFluidTraction(double current_time, unsigned int part = 0);
+    /*!
+     * Compute the exterior pressure used in the calculation
```

Comment:
Would it be better to interpolate the pressure in one function and the viscous stress in a separate functions? Sometimes we want only one or the other, right?

Reply:
So currently if the pressure jump is on, then the pressure on the structure is automatically calculated via interpolatePressureForTraction. Similarly if velocity jump is on, the WSS is computed automatically too (in the interpolateVelocity). There is a separate flag in the input file that asks to whether to compute the traction or not via computeFluidTraction